cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -34,8 +34,6 @@
     "development_stage": DEV_STAGE_AUTO_MIGRATE_MAP,
     "disease": {
     },
-    "organism": {
```

Comment:
i know we'll need to rewrite the migrate script, but planning to do that in a separate PR since this one is already a lot

Reply:
You'll want to remove this from the migrate script [template](https://github.com/chanzuckerberg/single-cell-curation/blob/main/scripts/migration_assistant/migration_template.jinja#L36) rather than the migrate script directly

Reply:
ah thanks for the call out, updated!